File: src/main/java/com/distribution/modules/api/pojo/vo/WithdrawalVo.java
package com.distribution.modules.api.pojo.vo;

import lombok.Data;

import javax.validation.constraints.NotNull;
import java.math.BigDecimal;

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 * @Description TODO(描述)
 * @create 2018/6/1-00:13
 */
@Data
public class WithdrawalVo {
    /**
     * 提现订单号
     */
    private String withdrawNum;
    /**
     * 提现状态 0:失败,1:成功
     */
    private String withdrawType;
    /**
     * 提现人姓名
     */
    private String withdrawName;
    /**
     * 手机号
     */
    @NotNull
    private String withdrawMobile;
    private String withdrawCard;
    /**
     * 金额
     */
    @NotNull
    private BigDecimal withdrawAmount;
    /**
     * 手续费
     */
    private BigDecimal withdrawPoundage = new BigDecimal(0);
    /**
     * 实际到账金额
     */
    private BigDecimal realAmount;
    /**
     * 支付宝账户
     */
    private String aliPayAccount;
}

File: src/main/kotlin/com/distribution/modules/api/pojo/vo/WithdrawalVo.kt
package com.distribution.modules.api.pojo.vo

import java.math.BigDecimal
import javax.validation.constraints.NotNull

/**
 * @author ChunLiang Hu
 * @Company
 * @Project distribution
 * @Package com.distribution.modules.api.pojo.vo
 
 * @create 2018/6/1-00:13
 */

data class WithdrawalVo(
        /**
         * 提现订单号
         */
        var withdrawNum: String = "",
        /**
         * 提现状态 0:失败,1:成功
         */
        var withdrawType: String = "",
        /**
         * 提现人姓名
         */
        var withdrawName: String = "",
        /**
         * 手机号
         */
        @NotNull
        var withdrawMobile: String = "",
        var withdrawCard: String = "",
        /**
         * 金额
         */
        @NotNull
        var withdrawAmount: BigDecimal = BigDecimal.ZERO,
        /**
         * 手续费
         */
        var withdrawPoundage: BigDecimal = BigDecimal(0),
        /**
         * 实际到账金额
         */
        var realAmount: BigDecimal = BigDecimal.ZERO,
        /**
         * 支付宝账户
         */
        var aliPayAccount: String = ""

)
